public/app/plugins/datasource/prometheus/configuration/AzureAuthSettings.tsx

```diff
@@ -11,9 +11,7 @@ import { AzureCredentialsForm } from './AzureCredentialsForm';
 export const AzureAuthSettings = (props: HttpSettingsBaseProps) => {
   const { dataSourceConfig, onChange } = props;
 
-  const [overrideAudienceAllowed] = useState<boolean>(
-    config.featureToggles.prometheusAzureOverrideAudience || !!dataSourceConfig.jsonData.azureEndpointResourceId
-  );
+  const [overrideAudienceAllowed] = useState<boolean>(!!dataSourceConfig.jsonData.azureEndpointResourceId);
```

Comment:
I think we can remove this so the option to override audience is always displayed and if we find out that it needs to be hidden the change can be reverted 😊

Reply:
I requested another review just to make sure everything is alright.